django/template/response.py

```diff
@@ -9,6 +9,9 @@ class ContentNotRenderedError(Exception):
 
 class SimpleTemplateResponse(HttpResponse):
     rendering_attrs = ["template_name", "context_data", "_post_render_callbacks"]
+    non_picklable_attrs = HttpResponse.non_picklable_attrs | frozenset(
+        ["template_name", "context_data", "_post_render_callbacks"]
+    )
```

Comment:
`rendering_attrs` are not used anymore, please remove them and add a short release note, e.g.
```
* The undocumented ``SimpleTemplateResponse.rendering_attrs`` and
  ``TemplateResponse.rendering_attrs`` are renamed to ``non_picklable_attrs``.
```
(see also https://github.com/django/django/pull/14728#discussion_r702713573).

Reply:
Ok 👍🏻